### DISCUSSION TITLE
Typo in readme

### DISCUSSION THREAD

**Discussion by JeffLiang2018** [golang-jwt/jwt#284] [General]
"go get -u github.com/golang-jwt/jwt/v5" doesn't work.
Instead "go get -u github.com/golang-jwt/jwt" can work

**Comment by mfridman** [comment]
What error are you getting? I just tried this:

```
go get -u github.com/golang-jwt/jwt/v5@latest
go: added github.com/golang-jwt/jwt/v5 v5.0.0-rc.1
```

This should download the [v5 release candidate 1](https://github.com/golang-jwt/jwt/releases/tag/v5.0.0-rc.1).

Fwiw with module on v2 and up, you'll always want to go get and import with the suffix included.
